### ISSUE TITLE
change URI of instances

### ISSUE THREAD

**Issue by liekeverhelst** [netwerk-digitaal-erfgoed/project-organisations-datasets#2]
The data instances should have a URI starting with:

https://data.netwerkdigitaalerfgoed.nl/registry

This should be extended with the class name and /id/ conforming to the [Dutch DSO URI strategy](https://aandeslagmetdeomgevingswet.nl/publish/pages/143597/uri-strategie_-_12_maart_2018.pdf)

So a dataset instance must have a URI like this:
https://data.netwerkdigitaalerfgoed.nl/registry/dataset/id/<qname>

A distribution instance like this:
https://data.netwerkdigitaalerfgoed.nl/registry/distribution/id/<qname>

etcetera

**Comment by roland-c** [comment]
I don't think it is a good idea to contain a class specification in an IRI for sake of simplicity and flexibility Those are the classes obtained no: http://lod.kb.nl/sparql?default-graph-uri=http%3A%2F%2Flod.kb.nl%2Fdatasets%2F&query=select+distinct+%3FConcept+where+%7B%5B%5D+a+%3FConcept%7D+LIMIT+100&should-sponge=&format=text%2Fhtml&timeout=180000&debug=on&run=+Run+Query+

Do we need to put every class in the URI spec?

**Comment by liekeverhelst** [comment]
I agree with your point of view but this is how it was agreed in NL. Let's leave it for the coming deadline and discuss in the future how we want to do this.

**Comment by roland-c** [comment]
Another consideration is the ID part of the IRI proposed above that is provided by the registry that is queried. It is possible that different registries provide the same ID for different datasets.  This would result in an error in the obtained dataset. The resolution to this is include an ID for the registry or repository in the IRI of a dataset preceding the ID for the dataset in the URI. 
Like this: http://data.netwerkdigitaalerfgoed.nl/registry/ID-Endpoint/ID-Dataset

**Comment by sdevalk** [comment]
@roland-c Yes, that's a valid use case. We can indeed include an ID of the originating data source in the URI. Alternatively, would it be a faster/simpler option to generate our own IDs, instead of having to rely on IDs of the source? This way we can guarantee the uniqueness of the dataset IDs.

**Comment by roland-c** [comment]
Yes, it may be necessary to maintain our own ID system, but I would rather count on the system of a registry. 
I changed the IRI composition for a Dataset or a Distribution of a dataset to http://data.netwerkdigitaalerfgoed/[ID-registry]/[ID-dataset|ID-distribution]

There is also the IRI for additional resources like Organisation, which are composed now like this:
http://data.netwerkdigitaalerfgoed/[ID-resource] 
But this should probably have an additional ID of the registry (?), like this:
http://data.netwerkdigitaalerfgoed/[ID-registry]/[ID-resource]

**Comment by roland-c** [comment]
URI's are now implemented as:
http://data.netwerkdigitaalerfgoed.nl/registry/ [ registryID ] / [ DatasetID | distributionID | organizationID | formatID ]
